ArcadiaParties-SPA/src/app/user/services/user.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { User } from '../models/User';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  public getUser() {
+    return {
+      user: {
+        identity: 'ekaterina.kuznetsova',
+        firstName: 'ekaterina',
+        lastName: 'kuznetsova',
+        department: 'horoshiy',
+        dates: [
+          { name: 'BirthDay', date: new Date('11.02.1944') },
+          { name: 'HireDay', date: new Date('03.12.1955') }
+        ],
+        roles: ['User', 'Admin']
+      }
+    };
```

Comment:
It changed

Reply:
This is also arguable, because Typescript implicitly infers correct return type. But, return type should be defined anyway, because otherwise, if you change the method and return type also changes, you will get errors in callers methods, not in the method itself.

Reply:
I still recommend explicit return type. As @sponkrashin said, we will be able to check if something go wrong.